ts/features/bonusVacanze/components/QrModalBox.tsx

```diff
@@ -122,6 +122,9 @@ const QrModalBox: React.FunctionComponent<Props> = (props: Props) => {
             style={styles.modalClose}
             onPress={onClose}
             transparent={true}
+            accessible={true}
+            accessibilityRole={"button"}
+            accessibilityLabel={I18n.t("global.buttons.close")}
```

Comment:
Could we use `ButtonDefaultOpacity` in place of `Button`?

Reply:
Yes it seems fine, changes will be in next commit